blocks/feed-newsroom/feed-newsroom.js

```diff
@@ -0,0 +1,195 @@
+import {
```

Comment:
Not sure if we need a new block or this could be an enhancement of the feed block itself. Let's discuss.

Reply:
Sure. Yeah its an enhancement of feed block with good enough changes. I do not want to alter feed block since that should have been leveraged at other places as well and hence created a new block rather.

Reply:
Ok then let's go with a separate block. Probably the parsers can be put at a common place to avoid code repetition.